pkg/util/testleak/add-leaktest.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
```

Comment:
Could you collect all scripts in a dedicated folder?

Reply:
addressed